src/main/java/io/zksync/methods/response/ZksBlock.java

```diff
@@ -0,0 +1,20 @@
+package io.zksync.methods.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.web3j.protocol.core.methods.response.EthBlock;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+
+public class ZksBlock extends EthBlock {
+
+
+    private String baseFeePerGas;
```

Comment:
`EthBlock` already contains `baseFeePerGas` property. I guess that there is no need to declare it in `ZksBlock` once again.